flytekit/core/constants.py

```diff
@@ -23,6 +23,9 @@
 # Set this environment variable to true to force the task to return non-zero exit code on failure.
 FLYTE_FAIL_ON_ERROR = "FLYTE_FAIL_ON_ERROR"
 
+# Set this environment variable to true to force pyflyte-fast-execute to run task-execute-cmd in a separate process
+FLYTE_FAST_EXECUTE_CMD_IN_NEW_PROCESS = "FLYTE_FAST_EXECUTE_CMD_IN_NEW_PROCESS"
+
```

Comment:
Could we run a workflow in the integration tests with `FLYTE_FAST_EXECUTE_CMD_IN_NEW_PROCESS` enabled? Just want to make sure the old behavior is still working.

https://github.com/flyteorg/flytekit/blob/99f6d9f76f093bae761adfe19a30ecb79bdf0177/tests/flytekit/integration/remote/test_remote.py#L105